crates/types/src/invocation.rs

```diff
@@ -65,6 +65,21 @@ pub enum MaybeFullInvocationId {
     Full(FullInvocationId),
 }
 
+impl MaybeFullInvocationId {
+    pub fn into_invocation_id(self) -> InvocationId {
+        match self {
+            MaybeFullInvocationId::Partial(iid) => iid,
+            MaybeFullInvocationId::Full(fid) => InvocationId::from(fid),
+        }
+    }
+}
```

Comment:
No need to add this method to duplicate the functionality of From/Into

Reply:
Will remove it.